soc/riscv/espressif_esp32/esp32c3/default.ld

```diff
@@ -276,7 +278,7 @@ SECTIONS
     * dram0_0_seg reflect the same address space on different buses.
     */
     . = ORIGIN(dram0_0_seg) + _iram_end - _iram_start;
-  } GROUP_LINK_IN(RAMABLE_REGION)
+  } > RAMABLE_REGION
```

Comment:
Those changes are pointless since the LD macro `GROUP_LINK_IN` expands the same.